ts/src/header-validator/trigger.test.ts

```diff
@@ -1136,6 +1136,22 @@ const testCases: jsontest.TestCase<Trigger>[] = [
       },
     ],
   },
+  {
```

Comment:
It’s easier to read with the test cases for the same field grouped together, but not quite consistent with the style of this file which separates valid test cases, warnings, and errors.